rgw/v2/tests/s3cmd/reusable.py

```diff
@@ -42,6 +44,27 @@ def create_bucket(bucket_name, ssl=None):
     assert expected_response in mb_response, error_message
 
 
+def create_versioned_bucket(user_info, bucket_name, ip_and_port, ssl=None):
```

Comment:
@TejasC88 , Is this method same as https://github.com/red-hat-storage/ceph-qe-scripts/blob/master/rgw/v2/tests/s3_swift/reusable.py#L427 ?